commit 493a2c2d23ca91afba96ac32b6cbafb54382c2a3
Author: Tom Lendacky <thomas.lendacky@amd.com>
Date:   Thu Feb 9 09:22:26 2023 -0600

    Documentation/hw-vuln: Add documentation for Cross-Thread Return Predictions
    
    Add the admin guide for the Cross-Thread Return Predictions vulnerability.
    
    Signed-off-by: Tom Lendacky <thomas.lendacky@amd.com>
    Message-Id: <60f9c0b4396956ce70499ae180cb548720b25c7e.1675956146.git.thomas.lendacky@amd.com>
    Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>

diff --git a/Documentation/admin-guide/hw-vuln/index.rst b/Documentation/admin-guide/hw-vuln/index.rst
index 4df436e7c417..e0614760a99e 100644
--- a/Documentation/admin-guide/hw-vuln/index.rst
+++ b/Documentation/admin-guide/hw-vuln/index.rst
@@ -18,3 +18,4 @@ are configurable at compile, boot or run time.
    core-scheduling.rst
    l1d_flush.rst
    processor_mmio_stale_data.rst
+   cross-thread-rsb.rst